src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -179,27 +336,38 @@ static void OnFlyoutBehaviorChanged(BindableObject bindable, object oldValue, ob
 			}
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='BackgroundColorProperty']/Docs/*" />
+		/// <summary>
+		/// Defines the background color in the Shell chrome. 
```

Comment:
This might stem from my own ignorance, but what's the Shell's chrome?

Reply:
Shell offers a basic navigation structure (tabs etc) that is what it refers to. In this case it would be the background color of that structure.
Appears multiple times in the Xamarin.Forms documentation https://learn.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/shell/pages